xen/arch/arm/domain.c

```diff
@@ -698,6 +702,27 @@ int arch_sanitise_domain_config(struct xen_domctl_createdomain *config)
     return 0;
 }
 
+#ifdef CONFIG_HOST_DTB_EXPORT
+#define HYPFS_PROPERTY_MAX_SIZE 4096
```

Comment:
I believe that DTB is much bigger than 4k. Also, naming is quite ambiguous. It just says "hypfs property". Which property exactly? 

I believe, this is the leftover from your previous version. But in this version it makes no sense